node/core/approval-voting/src/lib.rs

```diff
@@ -1180,7 +1180,7 @@ async fn handle_approved_ancestor(
 	target: Hash,
 	lower_bound: BlockNumber,
 	wakeups: &Wakeups,
-) -> SubsystemResult<Option<(Hash, BlockNumber)>> {
+) -> SubsystemResult<Option<(Hash, BlockNumber, Vec<(Hash, SessionIndex, Vec<CandidateHash>)>)>> {
```

Comment:
Perhaps a typedef for this would be useful?

Reply:
Added a full type, the tuple items deserve some description as well.